### PR TITLE
[8.0][FIX] project_task_materials_stock: Singleton unlink tasks

### PR DESCRIPTION
**Pull request by carlosdauden**

### CODE REVIEW

File: project_task_materials_stock/models/project.py [hunk 1]
```diff
@@ -18,15 +18,22 @@ class ProjectTaskType(models.Model):
...
-        self.analytic_line_ids = self.mapped('material_ids.analytic_line_id')
+        for task in self:
+            task.analytic_line_ids = task.mapped(
+                'material_ids.analytic_line_id')
 
+    @api.multi
     @api.depends('stock_move_ids.state')
-    def _check_stock_state(self):
+    def _compute_stock_state(self):
         if not self.stock_move_ids:
```
Comment (pedrobaeza) [inline project_task_materials_stock/models/project.py]: You have to iterate here over self.

**Comment by dreispt** [comment]
When @pedrobaeza comment if fixed, you have my +1

**Comment by pedrobaeza** [comment]
There's a lint error:

```
project_task_materials_stock/models/project.py:40:80: E501 line too long (80 > 79 characters)
```

**Comment by pedrobaeza** [comment]
:+1:

**Comment by sergio-teruel** [comment]
👍

**Comment by antespi** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Counting also @dreispt positive review after the change, I merge.

**Pull request merged**